DigitalLearningSolutions.Web/Views/LearningContent/Index.cshtml

```diff
@@ -29,9 +29,33 @@
 </div>
 
 @if (Model.Tutorials.Any()) {
-  <p class="nhsuk-u-margin-top-4">Course Tutorials</p>
+  <h2 class="nhsuk-heading-l">Course Tutorials</h2>
   <p class="nhsuk-body-m">Access to this content does not require login and progress is not tracked.</p>
   foreach (var tutorial in Model.Tutorials) {
     <partial name="_TutorialCard" model="tutorial" />
   }
 }
+
+@if (Model.Applications.Any()) {
+  <div id="@(Model.JavascriptSearchSortFilterPaginateEnabled ? "js-styling-hidden-area-while-loading" : "no-js-styling")">

```

Comment:
Yes, since the page has content above the paginated area that is outside the paginated area, and the paginated area may not always be included, I didn't include it as they overlap, and I don't think there will be a good way to guarantee the spinner always displays below the dynamic tutorials list.

Reply:
If I move it outside, then won't I not be able to have the conditional Courses section due to the page structure?